flowbased-computation/flowbased-computation-impl/src/main/java/com/farao_community/farao/flowbased_computation/impl/FlowbasedComputationImpl.java

```diff
@@ -56,6 +57,10 @@ public String getVersion() {
                 .withPtdfSensitivities(glsk, crac.getBranchCnecs(), Collections.singleton(Unit.MEGAWATT))
                 .build();
 
+        String initialNetworkId = network.getVariantManager().getWorkingVariantId();
+        network.getVariantManager().cloneVariant(initialNetworkId, "InitialStateWithPra");
+        network.getVariantManager().setWorkingVariant("InitialStateWithPra");
```

Comment:
Restore initial variant at the end of the computation

Reply:
Done